Masquerade4997Code/src/main/java/org/firstinspires/ftc/teamcode/Robots/MarkOne/Robot/MarkOne.java

```diff
@@ -25,31 +27,45 @@
     public MarkOneFoundationHook foundationHook;
     public MasqMotor  lift;
     public MasqMotorSystem intake;
-    public BNO055IMU imu;
     public DogeDetector detector;
 
     @Override
-    public void init(HardwareMap hardwareMap) {
+    public void mapHardware(HardwareMap hardwareMap) {
         driveTrain = new MasqMechanumDriveTrain(hardwareMap);
         blockGrabber = new MasqServo("blockGrabber", hardwareMap);
         lift = new MasqMotor("lift", MasqMotorModel.ORBITAL20, hardwareMap);
         blockRotater = new MasqServo("blockRotater", hardwareMap);
         intake = new MasqMotorSystem("intakeRight", DcMotorSimple.Direction.FORWARD, "intakeLeft", DcMotorSimple.Direction.REVERSE,MasqMotorModel.REVHDHEX40, hardwareMap);
         blockPusher = new MasqServo("blockPusher", hardwareMap);
-        imu = initializeIMU(imu, hardwareMap);
```

Comment:
What did you replace it with? In the MasqIMU class you should take whatever was in inializeIMU and put it in the constructor.